main.go

```diff
@@ -26,8 +26,13 @@ var yarnHistoryDir = flag.String("yarn-history-dir", "/tmp/staging/history/done"
 var httpTimeout = flag.Duration("http-timeout", time.Second*2, "The timeout used for connecting to YARN API. Pass values like: 2s")
 var pollInterval = flag.Duration("poll-interval", time.Second*5, "How often should we poll the job APIs. Pass values like: 2s")
 var enableDebug = flag.Bool("pprof", false, "Enable pprof debugging tools at /debug.")
+var s3BucketName = flag.String("s3-bucket", "stripe-data", "S3 bucket to fetch old jobs from")
+var s3Region = flag.String("s3-region", "us-west-2", "AWS region for the job storage S3 bucket")
+var s3JobsPrefix = flag.String("s3-jobs-prefix", "hadoop/job_history/json/job", "S3 key prefix (\"folder\") where jobs are stored")
+var s3FlowPrefix = flag.String("s3-flow-prefix", "hadoop/job_history/json/flow", "S3 key prefix (\"folder\") where cascading flows are stored")
 
 var jts map[string]*jobTracker
+var oldJobClient OldJobClient
```

Comment:
"oldJobClient" sounds like it's a deprecated class. i don't have a specific suggestion to replace it though.

Reply:
Yeaaaah I was worried about that too. it's meant to be `oldJobClient` as in "client for old jobs", as opposed to `recentJobClient` which is "client for recent jobs". i'll think of other names in the meantime